src/ProgressOnderwijsUtils.Analyzers/MaybeStateAnalyzer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace ProgressOnderwijsUtils.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class MaybeStateAnalyzer : DiagnosticAnalyzer
+    {
+        public static readonly DiagnosticDescriptor Rule = new DiagnosticDescriptor(
+            "POU2001",
+            "Maybe may not contain Maybe state",
+            "The maybe-error state may not be ignored.",

```

Comment:
Ik ben geen groot fan van "may not" want volgens mij is het ambigu.

Reply:
Wat stel jij voor?

Reply:
'must not' of 'is not allowed to'